test/strip/keep_section.txt

```diff
@@ -0,0 +1,34 @@
+;;; RUN: %(gen_wasm_py)s %(in_file)s -o %(temp_file)s.wasm
+;;; RUN: %(wasm-strip)s -k one %(temp_file)s.wasm -o %(temp_file)s_stripped.wasm
```

Comment:
Maybe spell out the full option name here (`--keep-section`) so the reader can clearly see what is being tests?

Reply:
```suggestion
;;; RUN: %(wasm-strip)s --keep-section one %(temp_file)s.wasm -o %(temp_file)s_stripped.wasm
```